### DISCUSSION TITLE
Markdown Syntax Throughout the Wiki

### DISCUSSION THREAD

**Discussion by bromiley** [forensicswiki/wiki#43] [General]
As we're reformatting and adding new content, I have a quick question about code blocks. Should we be using indentation or code blocks via backticks or tildes? The output will look the same but more consistent if some guidelines are set up front. Thanks!

**Comment by simsong** [comment]
I like spaces for a few lines of code and backticks for code blocks. I don't think that it really matters.

**Comment by joachimmetz** [comment]
I would opt fenced code blocks with triple back-ticks (```) and code type indicator, it looks like this is most widely adopted.

* https://www.mkdocs.org/user-guide/writing-your-docs/#fenced-code-blocks
* https://python-markdown.github.io/extensions/fenced_code_blocks/

**Comment by simsong** [comment]


**Reply by joachimmetz** [reply]
What "weird combination" (that is supported by mkdocs) did you have in mind? `<pre>...</pre>` ?

**Reply by simsong** [reply]
Well, DFXML is XML, and ` ``` ` is markdown.

**Reply by joachimmetz** [reply]
how does that relate to forensics wiki mkdocs style preference for code blocks?

**Reply by simsong** [reply]
I thought we were talking about DFXML. Sorry.

**Reply by joachimmetz** [reply]
Maybe delete this thread?

**Comment by joachimmetz** [comment]
Given no traction on this discussion moving forward with the following [pymarkdown configuration](https://github.com/forensicswiki/wiki/blob/main/pymarkdown.json). If there are strong reasons to change it, please open an issue to discuss further.

Closing this discussion.
